src/nanoemoji/config.py

```diff
@@ -23,14 +23,47 @@
 import toml
 from typing import Any, Iterable, MutableMapping, NamedTuple, Optional, Tuple, Sequence
 
-
-FLAGS = flags.FLAGS
+from nanoemoji import write_font
 
 
-_DEFAULT_CONFIG = "_default.toml"
+FLAGS = flags.FLAGS
 
 
-flags.DEFINE_string("config", _DEFAULT_CONFIG, "Config file")
+_DEFAULT_CONFIG_FILE = "_default.toml"
+
+# we use None as a sentinel for flag not set; FontConfig class has the actual defaults.
+# CLI flags override config file (which overrides default FontConfig).
+flags.DEFINE_string("config", None, "Config file")
+flags.DEFINE_integer("upem", None, "Units per em.")
+flags.DEFINE_string("family", None, "Family name.")
+flags.DEFINE_string("output_file", None, "Output filename.")
+flags.DEFINE_enum(
+    "color_format",
+    None,
+    sorted(write_font._COLOR_FORMAT_GENERATORS.keys()),
```

Comment:
I suggest you enumerate color formats in config.py, import that into write_font, and use it as the key to _COLOR_FORMAT_GENERATORS

Reply:
Done